packages/amplify_core/lib/src/types/storage/storage_bucket.dart

```diff
@@ -16,4 +16,9 @@ class StorageBucket {
 
   @internal
   BucketInfo resolveBucketInfo(StorageOutputs? storageOutputs) => _info;
+
+  @override
+  Map<String, Object?> toJson() => {
+        '_info': _info.toJson(),
+      };
```

Comment:
how about to use `bucket?.resolveBucketInfo.toJson()` for the storage options instead of adding toJson() to `StorageBucket` and to `StorageBucketFromOutputs`?